cli/bin/env-setup

```diff
@@ -6,12 +6,8 @@ else
     BIN_DIR=$PWD/bin
 fi
 
+# real, absolute path to BIN_DIR
 FULL_BIN_PATH=$(python -c "import os; print(os.path.realpath('$BIN_DIR'))")
 
+# ensure BIN_DIR is prepended to PATH
 expr "$PATH" : "${FULL_BIN_PATH}.*" > /dev/null || export PATH=$FULL_BIN_PATH:$PATH
-export DCOS_CONFIG=~/.dcos/dcos.toml
-
-if [ ! -f "$DCOS_CONFIG" ]; then
```

Comment:
Looks like this file is included by install.sh. Please move this creation block into `install-dcos-cli.sh` (looks like the Windows script already does this).